Content.Server/Xenoarchaeology/XenoArtifacts/Effects/Systems/PolyArtifactSystem.cs

```diff
@@ -0,0 +1,48 @@
+using Content.Server.Xenoarchaeology.XenoArtifacts.Effects.Components;
+using Content.Server.Xenoarchaeology.XenoArtifacts.Events;
+using Content.Shared.Humanoid;
+using Content.Server.Polymorph.Systems;
+using Content.Shared.Mobs.Systems;
+using Content.Shared.Polymorph;
+
+namespace Content.Server.Xenoarchaeology.XenoArtifacts.Effects.Systems;
+
+public sealed class PolyArtifactSystem : EntitySystem
+{
+    [Dependency] private readonly EntityLookupSystem _lookup = default!;
+    [Dependency] private readonly MobStateSystem _mob = default!;
+    [Dependency] private readonly PolymorphSystem _poly = default!;
+    [Dependency] private readonly SharedAudioSystem _audio = default!;
+
+    [ValidatePrototypeId<PolymorphPrototype>]
+    public const string PolymorphPrototypeName = "ArtifactMonkey";
```

Comment:
put this in PolyArtifactComponent instead of hardcoding it in the system.

Reply:
If i put it in component how do i retreive it in system? i tried component.PolymorphPrototypeName but it says i must use it as type.